dune/stuff/common/compiler.hh

```diff
@@ -7,14 +7,11 @@
 #define DUNE_STUFF_COMMON_COMPILER_HH
 
 #if defined(__GNUC__)
-# if defined(__GNUC_PATCHLEVEL__)
-#  define DS_GNUC_VERSION (__GNUC__ * 10000 \
-                            + __GNUC_MINOR__ * 100 \
-                            + __GNUC_PATCHLEVEL__)
-# else
-#  define DS_GNUC_VERSION (__GNUC__ * 10000 \
-                            + __GNUC_MINOR__ * 100)
-# endif
+#if defined(__GNUC_PATCHLEVEL__)
+#define DS_GNUC_VERSION (__GNUC__ * 10000 + __GNUC_MINOR__ * 100 + __GNUC_PATCHLEVEL__)
+#else
+#define DS_GNUC_VERSION (__GNUC__ * 10000 + __GNUC_MINOR__ * 100)
+#endif
 #endif
```

Comment:
I was very happy with our indentation of preprocessor directives but I could not find a way to let clang-format leave those alone.